WcaOnRails/config/locales/en.yml

```diff
@@ -23,20 +23,28 @@ en:
   common:
     here: "here"
     wca_id: "WCA ID"
-    search: "Search"
     all_regions: "All regions"
+  datepicker:
+    # Do *NOT* change these, they would break fields' validation
+    dformat: "%Y-%m-%d"
+    pformat: "YYYY-MM-DD"
+    dayViewHeaderFormat: "MMMM YYYY"
+  timepicker:
+    # Do *NOT* change these, they would break fields' validation
+    dformat: "%R"
+    pformat: "HH:mm"
```

Comment:
This is exactly the same for both fr and en and from the comment it looks like it should be the same for all language.
1. I'd suggest separate file for this.
2. Could we reuse this (file/locales) without copying for each language?